<launch>
  <!-- ONLY FOR TESTING -->
  <!-- REMOVE AFTER -->
  <node pkg="tf2_ros" type="static_transform_publisher"
    name="mount_link_to_base" args="0 0 0.0099 0 0 0 1 mount_link base"/>

  <node pkg="sawback_manipulation" type="grasp_detection" name="grasp_detection" output="screen">
    <remap from="cloud" to="/ridgeback/bumblebee/points2"/>
    <remap from="filtered_cloud" to="/ridgeback/grasps/filtered_cloud"/>

    <param name="remove_ground" value="false"/>
    <param name="cartesian_limits" value="false"/>

    <param name="filtered_cloud_frame" value="base"/>

    <rosparam param="transfrom_base_camera">[0.240, 0.001, 0.060, 1.0, 0.0, 0.0, 0.0]</rosparam>
    <rosparam param="transfrom_camera_optical">[0.01, 0.0, 0.0, 0.500, -0.500, 0.500, -0.500]</rosparam>

    <rosparam param="xyz_lower_limits">[-5.0, -5.0, 0.0]</rosparam>
    <rosparam param="xyz_upper_limits">[5.0, 5.0, 0.80]</rosparam>
  </node>

  <node name="$(anon rviz)" pkg="rviz" type="rviz" respawn="false"
          args="-d $(find sawback_manipulation)/config/sawback_grasps.rviz" output="screen">
  </node>
</launch>
